msteams-platform/concepts/resource-specific-consent.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Resource specific consent in Teams
+description: Describes resource specific consent to apps accessing Team's data and how to configure it.
+keywords: RSC resource specific consent
+---
+# Resource specific consent in Teams
+
+Resource-specific consent (RSC) in Microsoft Teams allows team owners to grant consent to apps to access their team's data and replaces the need for a global admin to approve the app with [tenant-wide admin permissions](~/azure/active-directory/manage-apps/grant-admin-consent).
+These accesses include the ability to create, rename and delete channels, read channel messages, create tabs, and add or remove members from a team. This article provides the step by step guide to configure RSC in your Teams app.
+
+![Consent screen.](../../assets/images/rsc/rsc-consentscreen.png)
+
+## Prerequisites (Admin)
+
+As an admin, you can control whether team owners in your organization can grant team-specific consent (enabled by default) through [settings](https://review.docs.microsoft.com/en-us/MicrosoftTeams/resource-specific-consent?branch=v-lanac-rsc) that you configure in the Azure Active Directory (Azure AD) PowerShell module or the Azure portal and the Microsoft Teams admin center.
+
+
+> **Important** : Changing any of these settings doesn't affect data access for apps that were already granted consent. 
+
+## Configure Teams App to support RSC (Developer)
+
+The basic steps required to configure a service and get a token from the Microsoft identity platform endpoint that your service can use to call Microsoft Graph are:
```

Comment:
This sentence is a bit clunky. I think it can be rewritten for greater clarity.

Reply:
I have referred an existing document here. We can still discuss this on call.

Reply:
Modified the statement.